doc/cephadm/services/rgw.rst

```diff
@@ -199,6 +199,41 @@ RGW daemons deployed for that RGW service. For example
     The daemon can still receive replication data unless it has been removed
     from the zonegroup and zone replication endpoints.
 
+Draining client connections on shutdown
+---------------------------------------
+
+When an RGW daemon is stopped by for any reason, including during the cephadm upgrade process,
+RGW offers a setting that will delay shutdown as the RGW daemon attempts to complete ongoing
+client requests. This setting is off by default but activated manually by either passing
+``--stop-timeout=<timeout-in-seconds>`` to the RGW process or by setting the
+``rgw_exit_timeout_secs`` config option for the RGW daemon. Cephadm can handle setting this
+up through the spec file by specifying the ``rgw_exit_timeout_secs`` parameter in the spec
```

Comment:
suggest "This value may be configured in the RGW service spec file by specifying ``rgw_exit_timeout_secs``.